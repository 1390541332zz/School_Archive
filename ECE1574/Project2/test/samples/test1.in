Data File: test/samples/test1.info
size average
