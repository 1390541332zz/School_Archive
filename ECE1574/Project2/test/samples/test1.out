Data file: test/samples/test1.info
Command: size average
	Sum: 30369629.00
	Country count: 81
	Average Size: 374933.69
